rapid/workflow/queue_handlers/handlers/ecs_queue_handler.py

```diff
@@ -153,14 +158,17 @@ def _run_task(self, task_definition):
             response_dict = ecs_client.run_task(**task_definition)
             if response_dict['failures']:
                 logger.error("Failures were found: [{}]".format(response_dict['failures']))
-                status_id = StatusConstants.FAILED
+                if 'limit' in response_dict['failures'][0]['reason']:
```

Comment:
just curious, do they have a code, or a http status they use that is specific? or is there only this error message?

Reply:
It is only this error message, unfortunately. There are no error types nor error codes. It is quite frustrating.